variables.tf

```diff
@@ -77,18 +77,18 @@ variable "group_role" {
   default     = "roles/editor"
 }
 
+variable "sa_roles" {
+  description = "A list of roles to give the default Service Account for the project (defaults to none)"
+  type        = list(string)
+  default     = []
+}
+
 variable "create_project_sa" {
   description = "Whether the default service account for the project shall be created"
   type        = bool
   default     = true
 }
 
-variable "sa_role" {
```

Comment:
To preserve backwards-compatibility, let's continue accepting `sa_role` on the "outer" modules but add a note to the description that it is deprecated.